webpack.system-addon.config.js

```diff
@@ -5,14 +5,14 @@ const absolute = relPath => path.join(__dirname, relPath);
 
 const resourcePathRegEx = /^resource:\/\/activity-stream\//;
 
-module.exports = {
+module.exports = (env = {}) => ({
   mode: "none",
   entry: absolute("content-src/activity-stream.jsx"),
   output: {
     path: absolute("data/content"),
     filename: "activity-stream.bundle.js",
   },
-  devtool: "source-map",
+  devtool: env.development ? "inline-source-map" : false,
```

Comment:
https://webpack.js.org/configuration/devtool/ seems to suggest using eval-source-map for development rather than inline-source-map.  What motivated the choice of inline-source-map?  If we do want to keep inline-source-map, I'd suggest adding a comment explaining the motivation.

Reply:
I am pretty sure `eval` is prevented by our CSP

Reply:
I'm mostly suggesting this because from that chart, it appears to slow down build times substantially, and our build times are already a bit slow.

We could use a different CSP for development.

That said, I don't have a crazy strong opinion here.  If it truly slows development enough, we can always switch it in the future.  So it's really up to you.

Reply:
Our existing implementation does regular production source maps, which are also slow (we never used eval) so I would consider that an enhancement

Reply:
OK, if you want to do that later, how about at least leaving `XXX switch to eval-source-map for faster builds` in the file, both as a reminder, and to give the reader a clue that there's no intrinsic reason we need to leave this as inline-source-map.

Reply:
The reason is that jar.mn doesn't have an entry to build/package the map file anymore. I suppose that should be in the comment for why it's inline…

Reply:
@Mardak that would also be true of eval-source-map, so I'm not sure I understand the point you're making.